tests/test_gcs.py

```diff
@@ -29,7 +29,9 @@ def test_ctr(self):
             self.assertIsNotNone(client._credentials)
 
     def test_annonymous_client(self):
-        # TODO: the following will not work with current monkey patching.
+        env = EnvironmentVarGuard()
+        env.set('KAGGLE_USER_SECRETS_TOKEN', 'foobar')
+        env.set('KAGGLE_KERNEL_INTEGRATIONS', 'GCS')
```

Comment:
I think you need to do `with env:` and init() to test this properly. This is probably still not the monkeypatched version.

Reply:
Done. Ops, mixed up.